cmd/nerdctl/image/image_list_test.go

```diff
@@ -96,10 +96,9 @@ LABEL version=0.1`, testutil.CommonImage)
 	defer base.Cmd("rmi", tempName).AssertOK()
 
 	busyboxGlibc, busyboxUclibc := "busybox:glibc", "busybox:uclibc"
-	base.Cmd("pull", busyboxGlibc).AssertOK()
+	base.Cmd("tag", testutil.CommonImage, busyboxGlibc).Run()
+	base.Cmd("tag", testutil.CommonImage, busyboxUclibc).Run()
```

Comment:
Needs a comment line to explain why not pull real images

Reply:
I also wonder if we can avoid using fake "busybox" names

Reply:
Yeah - I'll just use some other name instead - it doesn't really matter.

Reply:
> Needs a comment line to explain why not pull real images

Well, nothing is really "real" in our software world, right? :-) - jk :-)

I will write a comment.